### ISSUE TITLE
redux-saga/utils seemingly not found or installable

### ISSUE THREAD

**Issue by anjunatl** [clarketm/saga-monitor#3]
**Describe the bug**
I tried to add `saga-monitor` to my project following the instructions on the GitHub page, but got a Webpack error: 

```
 ERROR  Failed to compile with 1 errors                                                                                                                  16:17:25

This dependency was not found:

* redux-saga/utils in ./node_modules/@clarketm/saga-monitor/dist/saga-monitor.es.js

To install it, you can run: npm install --save redux-saga/utils
```

I tried running `npm install --save redux-saga/utils` and got:

```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/redux-saga/utils.git
npm ERR!
npm ERR! ERROR: Repository not found.
npm ERR! fatal: Could not read from remote repository.
```

I found [redux-saga-utils](https://www.npmjs.com/package/redux-saga-utils) on Google but it didn't appear to satisfy it.

I tried using a `webpack.config.js` setup as described in https://github.com/redux-saga/redux-saga/issues/894 but it didn't seem to work either, assuming I did it correctly.

**Additional context**

The react/redux/saga app lives in the context of Laravel & Laravel mix's webpack implementation and that could be causing some interference. If you think that could have something to do with it and have any ideas on things to look for, please let me know.

**Comment by anjunatl** [comment]
I'm not seeing anything related to utils with react-saga, is that something that used to be in their repo that has been removed or something? Still trying to debug this locally with no success so far.

**Comment by anjunatl** [comment]
react-saga/utils was moved in November, https://github.com/redux-saga/redux-saga/pull/1655

**Comment by clarketm** [comment]
@anjunatl – thanks for identification and investigation into this. It looks like `redux-saga/utils` was migrated to [`@redux-saga/testing-utils`](https://www.npmjs.com/package/redux-saga-testing). I will make the migration in `saga-monitor` to address this issue.

**Comment by clarketm** [comment]
I noticed that `redux-saga` just went through a major [v1.0.0](https://github.com/redux-saga/redux-saga/releases/tag/v1.0.0) release which explains the breaking behavior. Until I can get out a fix I have created a patch release [v1.0.9](https://github.com/clarketm/saga-monitor/releases/tag/v1.0.9) specifying a peer dependency on redux-saga ">=0.14.7 <1". 

It also might be worth checking out the [sagaMonitor](https://github.com/redux-saga/redux-saga/tree/master/examples/sagaMonitor) module in the redux-saga repo. This is where I adapted this from originally.
